src/style.css

```diff
@@ -0,0 +1,142 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  background-color: #e5e5e5;
+}
+
+.card {
+  display: flex;
```

Comment:
just add paddings to your card, and you don't need to add margins to each element to center it